executor/src/witgen/machines/fixed_lookup_machine.rs

```diff
@@ -120,14 +121,21 @@ impl IndexedColumns {
             )
             .0;
 
+        log::trace!(
+            "Done creating index. Size (as flat list): entries * (num_inputs * input_size + row_pointer_size) = {} * ({} * {} bytes + {} bytes) = {} bytes",
+            index.len(),
+            input_column_values.len(),
+            mem::size_of::<FieldElement>(),
```

Comment:
Is there a way to make this directly depend on the type of `index` (i.e. it changes when the type changes)?

Reply:
Make what depend?

Reply:
I would like to use something like `mem::size_of::<typeof::<index>::Value>()` here